allennlp/tests/semparse/worlds/atis_world_test.py

```diff
@@ -294,8 +294,6 @@ def test_atis_local_actions(self): # pylint: disable=no-self-use
                  'number -> ["5"]',
```

Comment:
Where do 5, 6, and 8 come from?

Reply:
`5` comes from `May`, and `8` comes from `August`. Strictly speaking, `6` should not be here but it is showing up because `twenty sixth` appears and it is added when checking each token --  it is triggered by `sixth`.